app/schema_loader/schema_loader.py

```diff
@@ -4,10 +4,6 @@
 
 from app import settings
 
-import boto3
```

Comment:
Actually, watchtower (which pushes to cloudwatch) uses botocore too, so there is a dependency there.